bundle/tests/scorecard/kuttl/test-iqe-jobs/05-assert-with-default.yaml

```diff
@@ -48,6 +48,12 @@ spec:
               value: "smoke"
             - name: IQE_FILTER_EXPRESSION
               value: "test_plugin_accessible"
+            - name: IQE_REQUIREMENTS
+              value: ''
```

Comment:
I think these are being asserted from a previous job hanging around with the vars filled out. You can update these to match the other tests, or update `05-pods-with-default.yaml` to check the empty values.